src/core/common/Sidebar/getMenuType.js

```diff
@@ -1,19 +1,40 @@
 import get from "lodash/get";
 
-export const SHORT_MENU = "short";
-export const FULL_MENU = "full";
+import { themeShortMenu, themeFullMenu } from "../../../version/config/theme.config";
 
-/**
- * @author Bogdan Shcherban <bsc@piogroup.net>
- * @param {string} currentPathname
- * @return {string}
- */
-export function getMenuType(currentPathname) {
+const defaultShortMenu = [
+    { url: '/', label: 'Charts' },
+    { url: '/patients', label: 'Patients' }
+];
+
+const defaultFullMenu = [
+    { url: '/summary', label: 'Patient Summary' },
+    { url: '/problems', label: 'Problems / Issues' },
+    { url: '/medications', label: 'Medications' },
+    { url: '/vaccinations', label: 'Vaccinations' },
+    { url: '/allergies', label: 'Allergies' },
+    { url: '/contacts', label: 'Contacts' },
+    { url: '/top3Things', label: 'TopThreeThings' },
+];
+
+function isUrlPresented(currentResource, menuItemsArray) {
+    const filterArray = menuItemsArray.filter(item => {
```

Comment:
instead of using filter and testing if the length is not null, why don't you use `Array.includes`?

Reply:
But is Array.includes() work with associative array?
I want to check "url" key in each array item. Can I do this by Array.includes()?

Reply:
Associative array is a PHP data structure. Javascript has arrays and objects. If you can use filter(), then it's an array, so you can use includes().

Reply:
Yes, I understand working principle of Array.includes(). But my array is:
```
const myArray = [
    { url: "/firstUrl", label: "First Label" },
    { url: "/secondUrl", label: "Second Label" },
];
```
If I want to check presence by this function, I should also add the label:
```
function isUrlPresented(currentResource, menuItemsArray) {
      return menuItemsArray.includes({ url: '/' + currentResource, label: ???????? });
}
```
So, I should firstly find a label. Is it rational approach?